Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -326,6 +331,64 @@ open class MapView: UIView {
             mapboxMap: mapboxMap)
     }
 
+    private func subscribeToLifecycleNotifications() {
+        if #available(iOS 13.0, *), bundle.infoDictionary?["UIApplicationSceneManifest"] != nil {
+            notificationCenter.addObserver(self,
+                                           selector: #selector(sceneWillEnterForeground(_:)),
+                                           name: UIScene.willEnterForegroundNotification,
+                                           object: window?.parentScene)
```

Comment:
As with `UIApplication.willEnterForegroundNotification`, `UIScene.willEnterForegroundNotification` is probably premature as well. `UIScene.didActivateNotification` would be more reliable.

Reply:
#1402 replaces this with `UIApplication.didBecomeActiveNotification`.